tools/proofers/previewControl.js

```diff
@@ -299,7 +326,8 @@ window.addEventListener("DOMContentLoaded", () => {
         localStorage.preview_data = JSON.stringify(previewStyles);
     }
 
-    previewControl = {
+    // This is a hack to make previewControl global outside this module
+    window.previewControl = {
```

Comment:
`previewControl` is effectively an export of this module, but it's used by in-page JS which can't `import` it. Since how we use this module will change with the new proofreading interface this seems good enough for now.

Reply:
You could use the dynamic import: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import
This allows importing modules from non module environments. You'd have to make it a promise.

Reply:
It wouldn't help at all, so let's not bother.